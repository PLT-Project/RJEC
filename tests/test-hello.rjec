func main() {
  prints("hello world");
  printi(321);
  printi(888);
  printb(true);
  printb(false);
  printc('r');
  printc('J');
  printc('e');
  printc('C');
}
