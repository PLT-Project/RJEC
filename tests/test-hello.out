hello world
321
888
1
0
r
J
e
C
